src/static/css/cfpb-chart-builder.less

```diff
@@ -60,18 +60,38 @@
     .chart-label();
   }
 
+  .highcharts-label__tile-map {
+    .chart-label();
+    color: @black;
+    font-family: 'AvenirNextLTW01-Demi',Arial,sans-serif;
```

Comment:
Can this use `.u-webfont-demi()` from CF? Not sure what the dependency setup is here and whether that is available.